tests/compute/default-initializer.slang

```diff
@@ -0,0 +1,35 @@
+// default-initializer.slang
+
+// Confirm that a type with a default initializer gets
+// default-initialized where appropriate.
+
+//TEST(compute):COMPARE_COMPUTE:
+//TEST(compute):COMPARE_COMPUTE:-cpu
+
+struct Stuff
+{
+	int value;
```

Comment:
The tabbing looks odd here.

Reply:
I think the editor I'm using to write tests is inserting actual tab characters, and I sometimes end up with a mix of tabs and spaces as a result. At this point it might be best to have a standalone PR that just does a full tabs->spaces pass on all the test files.